app/views/katello/subscription_mailer/subscription_expiry.text.erb

```diff
@@ -0,0 +1,13 @@
+<%= _("Subscriptions expiring soon") %>
+
+<% @pools.each do |pool| %>
+  <%= pool %>:
+  <%= _("Subscription Name") %>: <%= pool.subscription.name %>
+  <%= _("Organization") %>:      <%= Organization.find_by_id(pool.organization_id).name %>
+  <%= _("Type") %>:              <%= pool.pool_type %>
+  <%= _("Quantity") %>:          <%= pool.quantity %>
+  <%= _("SKU") %>:               <%= pool.subscription.cp_id %>
+  <%= _("Contract") %>:          <%= pool.contract_number %>
+  <%= _("Account Start") %>:     <%= pool.start_date %>
```

Comment:
let's change Account Start and Account End to 'Start Date' and 'End Date' respectively